### PR TITLE
Order forms

### PR DESCRIPTION
**Pull request by jgoley**
Allows users to sort forms by dragging and dropping.
Also sets the order properties of forms and questions on creation.

I realized we will need to update the order properties of questions and forms on delete. I've added a bug to PT for questions: https://www.pivotaltracker.com/story/show/117961163

### CODE REVIEW

File: app/server/methods.coffee [hunk 1]
```diff
@@ -10,10 +10,12 @@ Meteor.methods
 
   createForm: (surveyId, props)->
     #TODO Authenticate
+    currentFormCount = getSurveys().findOne(surveyId).forms.length or 0
     formId = getForms().insert
       name: props.name
       createdBy: @userId
       questions: []
+      order: ++currentFormCount
```
Comment (nathanathan) [inline app/server/methods.coffee]: It seems like we could avoid updating the order values of all the forms when one is deleted by setting the order of new forms to 1 + the order value of the last form.

Comment (jgoley) [inline app/server/methods.coffee]: Looks like I may have been premature with the statement about updating `order` on delete. Looks like the sortable package handles re-ordering well when there is a gap in values.

Comment (jgoley) [inline app/server/methods.coffee]: Oh, I see...we still need to make that change.

**Comment by nathanathan** [comment]
:+1:

**Pull request merged**